src/backend/distributed/sql/udfs/citus_nodes/latest.sql

```diff
@@ -0,0 +1,38 @@
+CREATE VIEW public.citus_nodes AS
+SELECT
+  nodename AS "Host",
+  nodeport AS "Port",
+  CASE WHEN groupid = 0 THEN 'coordinator' ELSE 'worker' END AS "Node Type",
+  pg_size_pretty(citus_node_database_size(nodename, nodeport)) AS "Database Size",
+  (SELECT
+     count(*)
+   FROM
+     pg_dist_partition t
+   JOIN
+     pg_dist_shard s
+   USING
+     (logicalrelid)
+   JOIN
+     pg_dist_placement p
+   USING
+     (shardid)
+   WHERE p.groupid = n.groupid AND t.partmethod <> 'n'
```

Comment:
Should we also check repmodel here otherwise citus_local tables will also be excluded

Reply:
we're counting shards here, so citus local should definitely be excluded.